lib/src/components/NxDropdownMenu/__tests__/NxDropdownMenu.test.tsx

```diff
@@ -5,25 +5,19 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { mount } from 'enzyme';
-import 'jest-enzyme';
 
-import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import { render, within } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 import NxDropdownMenu, { Props } from '../NxDropdownMenu';
 
 describe('NxDropdownMenu', function() {
```

Comment:
Despite its name beginning with `Nx` (it probably shouldn't) this component is not publicly exposed and so shouldn't have tests.  The external behavior that is relevant to the two tests here is already covered in `NxDropdown`'s tests (e.g. the focus moving to the toggle when the dropdown is closed)

Reply:
> Despite its name beginning with `Nx` (it probably shouldn't) this component is not publicly exposed and so shouldn't have tests. The external behavior that is relevant to the two tests here is already covered in `NxDropdown`'s tests (e.g. the focus moving to the toggle when the dropdown is closed)

Removed it here: 430b2eb649c2cae5205dfa78e920f94ff10049dc